Loadouts/csat_ctar_pacmech_apex.hpp

```diff
@@ -0,0 +1,298 @@
+// Author: BWMF
+// Description: CSAT: Pacific Mechanized WARNING: Spez is replaced with VIPER team, reducing role varity. Verify slot names correct before play
+
+#include "undef.hpp" // Reset defines
+
+// Rifle
+#define RIFLE "arifle_CTAR_blk_F","arifle_CTAR_ghex_F"
+#define RIFLE_MAG "30Rnd_580x42_Mag_F:8","30Rnd_580x42_Mag_Tracer_F:2"
+#define RIFLE_ATTACHMENTS "acc_pointer_IR","optic_ACO_grn"
+#define AAR_ATTACHMENTS RIFLE_ATTACHMENTS
+// GL Rifle
+#define GLRIFLE "arifle_CTAR_GL_blk_F"
+#define GLRIFLE_MAG RIFLE_MAG
+#define GLRIFLE_MAG_SMOKE "1Rnd_Smoke_Grenade_shell:2","1Rnd_SmokeRed_Grenade_shell:2"
+#define GLRIFLE_MAG_HE "1Rnd_HE_Grenade_shell:5"
+#define GLRIFLE_MAG_FLARE "UGL_FlareRed_F:4"
+// Carbine
+#define CARBINE "arifle_Katiba_F"
+#define CARBINE_MAG "30Rnd_65x39_caseless:8","30Rnd_65x39_caseless_green:2"
+// AR
+#define AR "arifle_CTARS_blk_F","arifle_CTARS_ghex_F"
+#define AR_MAG "100Rnd_580x42_Mag_F:6"
+// Recon Rifle Attachments
+#define RECONRIFLE_ATTACHMENTS  "rhs_acc_tgpa","rhs_acc_2dpZenit","rhs_acc_pkas"
+// AT
+#define AT "launch_RPG32_ghex_F"
+#define AT_MAG "RPG32_F:3","RPG32_HE_F:1"
+// MMG
+#define MMG "MMG_01_tan_F"
+#define MMG_MAG "150Rnd_93x64_Mag:5"
+// MAT
+#define MAT "launch_O_Titan_short_ghex_F"
+#define MAT_MAG "Titan_AT:4"
+#define MAT_MAG2 "Titan_AT:3"
```

Comment:
I wouldn't really expect this loadout to be used against RHS vehicles, but I imagine probably not well.